desc/integrals/_interp_utils.py

```diff
@@ -0,0 +1,754 @@
+"""Fast interpolation utilities.
+
+Notes
+-----
+These utilities are chosen for performance on gpu among
+methods that have the best (asymptotic) algorithmic complexity.
+For example, we prefer to not use Horner's method.
+"""
+
+from functools import partial
+
+import numpy as np
+from interpax import interp1d
+from orthax.chebyshev import chebroots
+
+from desc.backend import dct, jnp, rfft, rfft2, take
+from desc.integrals._quad_utils import bijection_from_disc
+from desc.utils import Index, errorif, safediv
+
+# TODO:
```

Comment:
are these also documented in an issue already? just so it does not get lost in the code

Reply:
this one doesn't have an issue yet; the others are more important. right now we use cubic splines generated from our fourier chebyshev $\theta$ to compute bounce points.